resources/documentation/install_first_management.rst

```diff
@@ -121,11 +144,11 @@ Repositories structure follows a specific pattern:
 
                   Distribution    Version   Architecture    Repository
                         +             +       +               +
-                        |             +---+   |               |
-                        +-----------+     |   |      +--------+
-                                    |     |   |      |
-                                    v     v   v      v
-       /var/www/html/repositories/centos/7.6/x86_64/os
+                        |             +--+    |               |
+                        +-----------+    |    |    +----------+
+                                    |    |    |    |
+                                    v    v    v    v
+       /var/www/html/repositories/centos/7/x86_64/os
```

Comment:
Would be nice to explain here the idea of splitting image and packages.
And maybe move this explanation part before all the iso copy procedure.

Reply:
Done